resources/provider/migrations/postgres/5_v0.5.0.up.sql

```diff
@@ -32,3 +32,19 @@ CREATE TABLE IF NOT EXISTS "azure_redis_services" (
 	CONSTRAINT azure_redis_services_pk PRIMARY KEY(subscription_id,id),
 	UNIQUE(cq_id)
 );
+
+-- Resource: sql.servers
```

Comment:
The migrations tests are not supporting filenames called unreleased so we have to put all unreleased migrations into the next version file